release_notes/44-release-notes.adoc

```diff
@@ -563,6 +565,44 @@ The issue occurred when more than 10 collections were created and a vulnerabilit
 +
 This update ensures that the linked collection is retained, avoiding the error message.
 
+[id="resolved-in-version-441_{context}"]
+=== Resolved in version 4.4.1
+
+*Release date*: 22 April 2024
+
+* Fixed an issue that prevented the filtering of newly added or updated policies by category by using the `roxctl` CLI.
+
+[id="resolved-in-version-442_{context}"]
+=== Resolved in version 4.4.2
+
+*Release date*: 15 May 2024
+
+This release provides the following bug fixes:
+
+* Before this update, Collector pods on nodes with 128 or more cores would fail with a `CrashLoopBackOff` status due to issues with how the CO-RE BPF allocated kernel memory. The patch release fixes this issue.
```

Comment:
```suggestion
* Before this update, collector pods on nodes with 128 or more cores would fail with a `CrashLoopBackOff` status due to issues with how the CO-RE BPF allocated kernel memory. The patch release fixes this issue.
```